source/blog/2016-02-06-rspec-3-5-0-beta1-has-been-released.md

```diff
@@ -0,0 +1,44 @@
+---
+title: RSpec 3.5.0.beta1 has been released!
+author: Sam Phippen
+---
+
+RSpec 3.5.0.beta1 has just been released! This is a big release for us because
+it's the first release of RSpec we've published that has anything that could be
+described as approximating support for Rails 5.
+
+The purpose of this beta release is to enable people who are upgrading to Rails
+5 to check their existing RSpec suites with Rails 5. If you do find any problems
+with this release, let us know via GitHub, and we'll hopefully get everything
+fixed in time for the Rails 5 release proper.
+
+## Upgrading smoothly
+
+As was discussed at Railsconf this year, Rails "soft deprecated" controller
+tests in Rails 5. RSpec has been affected by being downstream of this, but
+fortunately, we were able to make the process relatively smooth for our users.
+
+If you have existing specs with `:type => :view` or `:type => :controller`
```

Comment:
It's slightly confusing that Rails' deprecation of _controller_ tests affect _view_ specs.  Not sure if it's worth spending any effort explaining why but I found this surprising.